content/en/docs/concepts/overview/working-with-objects/common-labels.md

```diff
@@ -0,0 +1,149 @@
+but aren't required for any core tooling.
+{{< /note >}}
+
+Shared labels and annotations share a common prefix: `app.kubernetes.io`. Labels
+without a prefix are private to users. The shared prefix ensures that shared labels
+do not interfere with custom user labels.
+
+## Labels
+
+In order to take full advantage of using these labels, they should be applied
+on every resource object.
+
+| Key                                 | Description           | Example  | Type |
+| ----------------------------------- | --------------------- | -------- | ---- |
+| `app.kubernetes.io/name`            | The name of the application | `mysql` | string |
+| `app.kubernetes.io/version`         | The current version of the application (e.g., a semantic version, revision hash, etc.) | `5.7.21` | string |
+| `app.kubernetes.io/component`       | The component within the architecture | `database` | string |
+| `app.kubernetes.io/part-of`         | The name of a higher level application this one is part of | `wordpress` | string |
+| `app.kubernetes.io/managed-by`  | The tool being used to manage the operation of an application | `helm` | string |
+
+To illustrate these labels in action, consider the following StatefulSet object:
+
+```yaml
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  labels:
+    app.kubernetes.io/name: mysql
+    app.kubernetes.io/version: "5.7.21"
+    app.kubernetes.io/component: database
+    app.kubernetes.io/part-of: wordpress
+    app.kubernetes.io/managed-by: helm
+```
+
+## Examples
+
+To illustrate different ways to use these labels the following examples have varying complexity.
+
+### A Simple Stateless Service
+
+Consider the case for a simple stateless service deployed using `Deployment` and `Service` objects. The following two snippets represent how the labels could be used in their simplest form.
+
+The `Deployment` is used to oversee the pods running the application itself.
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/name: myservice
+...
+```
+
+The `Service` is used to expose the application.
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app.kubernetes.io/name: myservice
+...
+```
+
+### Web Application With A Database
+
+Consider a slightly more complicated application: a web application (WordPress)
+using a database (MySQL), installed using Helm. The following snippets illustrate
+the start of objects used to deploy this application.
+
+The start to the following `Deployment` is used for WordPress:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/name: wordpress
+    app.kubernetes.io/version: "4.9.4"
+    app.kubernetes.io/managed-by: helm
+...
+```
+
+The `Service` is used to expose WordPress:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app.kubernetes.io/name: wordpress
+    app.kubernetes.io/version: "4.9.4"
+    app.kubernetes.io/managed-by: helm
```

Comment:
Recommend adding labels:
`app.kubernetes.io/component: server`
`app.kubernetes.io/part-of: wordpress`